### PR TITLE
Rhd 1007

### PR DESCRIPTION
**Pull request by RubyGemTSL**
Complete work for Products RHD-1007.

### CODE REVIEW

**Comment by paulrobinson** [comment]
@ollyjshaw can you review please

File: _cucumber/lib/pages/products_page.rb [hunk 1]
```diff
@@ -21,10 +37,17 @@ def open
   end
 
   def product_titles
-     titles = []
-     elements = [infrastructure_management, cloud_products,mobile, jboss_dev_and_management, integration_and_automation]
-     elements.each { |el| titles << el.text }
-    titles
+    t = []
```
Comment (ollyjshaw) [inline _cucumber/lib/pages/products_page.rb]: was better as titles

File: _layouts/product-index.html.slim [hunk 2]
```diff
@@ -1,61 +1,60 @@
 ---
```
Comment (ollyjshaw) [inline _layouts/product-index.html.slim]: With this file it looks like everything has been indented by two spaces. Can we clear this so we can see more clearly what the changes were?

**Comment by ollyjshaw** [comment]
Have we tested this inside of docker and as a rake task?

**Comment by RubyGemTSL** [comment]
@ollyjshaw - yeah all passed in local docker. just addressing comments and ran them again. Updates to follow

**Comment by ollyjshaw** [comment]
@RubyGemTSL great, :+1: when you've done with comments.

**Comment by RubyGemTSL** [comment]
@ollyjshaw - I'm done with comments.

I dont know whats going on with the indentation of the product-index.html.slim page. I took a direct copy just now and re-added the id's and still happening. For this page I added the id's that are now there.

**Comment by paulrobinson** [comment]
Waiting for whitespace changes to be reverted.

**Comment by paulrobinson** [comment]
Can you also squash the commits when you are done. If you've not done this before, I suggest you take a copy of your local repo as you can do permanent damage to your branch.

**Comment by RubyGemTSL** [comment]
@paulrobinson @ollyjshaw - should be good now?

**Comment by RubyGemTSL** [comment]
@redhat-developer-ci run the tests

**Pull request merged**